scala/pom.xml

```diff
@@ -0,0 +1,116 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
It is a fake module only used to support different scala version in one build, you could refer to @vanzin 's comment and here https://github.com/vanzin/multi-scala .